deno.json

```diff
@@ -1,3 +1,4 @@
 {
-  "unstable": ["byonm"]
+  "unstable": ["byonm"],
+  "lock": false
```

Comment:
The important packages used in prod are managed by `bun` and it has a lockfile already.
The libs used for testing aren't important enough to lock, IMHO. It's not a big bother having the file, but less files in the repo root is generally preferred.